docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Scale Azure OpenAI for .NET chat sample using RAG
+description: Learn how to add load balancing to your application to extend the chat app beyond the Azure OpenAI token and model quota limits.
+ms.date: 03/06/2024
+ms.topic: get-started
+ms.custom: devx-track-java, devx-track-java-ai
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my OpenAI capacity to avoid rate limit errors.
+---
+
+# Scale Azure OpenAI for .NET chat using RAG with Azure Container Apps
+
+[!INCLUDE [aca-load-balancer-intro](~/azure-dev-docs-pr/articles/intro/includes//scaling-load-balancer-introduction-azure-container-apps.md)]
+
+## Prerequisites
+
+* Azure subscription.  [Create one for free](https://azure.microsoft.com/free/ai-services?azure-portal=true)
+* Access granted to Azure OpenAI in the desired Azure subscription.
+
+    Currently, access to this service is granted only by application. You should [apply for access](https://aka.ms/oai/access) to Azure OpenAI.
+
+* [Dev containers](https://containers.dev/) are available for both samples, with all dependencies required to complete this article. You can run the dev containers in GitHub Codespaces (in a browser) or locally using Visual Studio Code.
+
+#### [Codespaces (recommended)](#tab/github-codespaces)
+
+* GitHub account
```

Comment:
This looks a bit odd to me as a single bullet by itself - should we change this to say something like "Only a GitHub account is required to use CodeSpaces"